build.sbt

```diff
@@ -11,7 +11,7 @@ scalaVersion := "2.10.4"
 scalacOptions += "-feature"
 
 libraryDependencies ++= Seq(
-  "org.webjars" % "rjs" % "2.1.15"
+  "org.webjars" % "requirejs-node" % "2.1.17"
```

Comment:
My PR was merged and a new release cut, so `"org.webjars" % "rjs" % "2.2.20"` should work now as well